docs/docs/getting-started/first-steps-local.md

```diff
@@ -4,19 +4,25 @@
 With the `constellation mini` command, you can deploy and test Constellation locally without a cloud subscription. This mode is called MiniConstellation. Conceptually, MiniConstellation is similar to [MicroK8s](https://microk8s.io/), [K3s](https://k3s.io/), and [minikube](https://minikube.sigs.k8s.io/docs/).
 <!-- vale on -->
 
-MiniConstellation uses virtualization to create a local cluster with one control-plane node and one worker node. 
+MiniConstellation uses virtualization to create a local cluster with one control-plane node and one worker node.
 
 :::info
 
-MiniConstellation **doesn't** require hardware with Confidential VM (CVM) support. 
+MiniConstellation **doesn't** require hardware with Confidential VM (CVM) support.
 For attestation, MiniConstellation currently uses the software-based vTPM provided by KVM/QEMU.
 
 :::
 
+:::warning
```

Comment:
docusaurus doesn't know *warning*. Choose one of these https://docusaurus.io/docs/markdown-features/admonitions